nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -94,6 +94,8 @@ Available as [services.patroni](options.html#opt-services.patroni.enable).
 
 ## Backward Incompatibilities {#sec-release-22.11-incompatibilities}
 
+- nixpkgs now requires nix 2.3 or newer.
```

Comment:
```suggestion
- Nixpkgs now requires Nix 2.3 or newer.
```

Yes I'm being anal about capitalisation :-)

Reply:
Thanks, I didn't know that Nixpkgs and Nix are to be capitalized. Learned something along the way ;)